src/Fhir.Anonymizer.Shared.Core/AnonymizationConfigurations/AnonymizationFhirPathRule.cs

```diff
@@ -15,10 +16,13 @@ public class AnonymizationFhirPathRule : AnonymizerRule
 
         public string ResourceType { get; private set; }
 
+        public string ReplaceWith { get; set; }
+        // The replacement value is primitive or complex
+        public bool IsPrimitiveReplacement { get; set; }
         // Path is same to the resource type
         public bool IsResourceTypeRule { get { return Path.Equals(ResourceType);  } }
 
-        public static AnonymizationFhirPathRule CreateAnonymizationFhirPathRule(Dictionary<string, string> config)
+        public static AnonymizationFhirPathRule CreateAnonymizationFhirPathRule(Dictionary<string, JToken> config)
```

Comment:
Could we double confirm jtoken for array? Maybe we can just use object instead. And let processor parse the object.

Reply:
Though JToken works for JString, Jarray, Jobject. Considering if we support XML/yaml configuration format, object is a good idea and I have changed to object